### ISSUE TITLE
[Notion] Add the ability to pass the database as an argument in Create Database Page

### ISSUE THREAD

**Issue by RedSlowpoke** [raycast/extensions#11295]
### Extension

https://www.raycast.com/notion/notion

### Description

Current Situation:
Currently, users of Raycast must manually select the database each time they create a quicklink, which can be time-consuming and prone to errors.

Proposed Enhancement:
Introduce the ability to pass arguments when creating a quicklink in Raycast. This enhancement will enable users to create multiple quicklinks directly to desired databases, streamlining the workflow and eliminating repetitive manual selections.

Benefits:

Efficiency: Users can specify the desired database as an argument, bypassing the need for manual selection.
Accuracy: Minimize the risk of selecting the wrong database by directly specifying it in the quicklink creation process.
Productivity: Enable users to create quicklinks for multiple databases swiftly, enhancing overall productivity.

### Who will benefit from this feature?

The ones who often generate pages in the same database, but have more than 1 in the workspace.

**Comment by dariuszpaluch** [comment]
@RedSlowpoke I think it's working like you want already.  You need to select default values and database, then click Create Deeplink, save it and when you open deeplink the values will be selected.

Take a look on video here https://github.com/raycast/extensions/pull/11178
This is my PR with ability to hide some fields and change order in your form, so on video I hide database field because it will take from Deeplink arguments.

**Comment by RedSlowpoke** [comment]
Indeed! I'm newbie to the Raycast, need to wrap my head around how those things working.
Thank you @dariuszpaluch

**Comment by dariuszpaluch** [comment]
@RedSlowpoke Tip: If you would like to change icon for your deeplink, then you need to create custom script command instead of deeplink 😃
